### PR TITLE
Docs: postgres functions

### PR DESCRIPTION
**Pull request by kiwicopple**
Starts the new guide for Postgres Functions:

- Functions: https://docs-git-docs-functions-supabase.vercel.app/docs/guides/database/functions

### CODE REVIEW

**Review by awalias — APPROVED** [review]
the language switchers are 🔥

File: web/docs/guides/database/functions.mdx [hunk 1]
```diff
@@ -0,0 +1,312 @@
...
+  height="385"
+  src="https://www.youtube-nocookie.com/embed/MJZCCpCYEqk"
+  frameBorder="1"
+  allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture"
+  allowFullScreen
+></iframe>
+
+## Creating Functions 
+
+Supabase provides several options for creating database functions. You can use the Table Editor or create them directly using SQL.
```
Comment (awalias) [inline web/docs/guides/database/functions.mdx:28]: "dashboard" instead of "Table Editor"?

**Comment by awalias** [comment]
this might be out of scope for this guide but the thing most people find confusing is which bits of the function code go into the function definition text box in the dashboard:
image

File: web/docs/guides/database/functions.mdx [hunk 2]
```diff
@@ -0,0 +1,312 @@
...
+```
+
+</TabItem>
+<TabItem value="JS">
+
+```js 
+const { data, error } = supabase
+  .rpc('get_planets')
+  .eq('id', 1)
+```
```
Comment (bnjmnt4n) [inline web/docs/guides/database/functions.mdx:223]: I think you might have missed the `.select("name")` call to select just the name field like the description specifies.

**Comment by kiwicopple** [comment]
> into the function definition text box

Yeah I deliberately left this out because that whole section needs to be redone. It's a bit difficult to use right now

**Pull request merged**